### ISSUE TITLE
Map rotation

### ISSUE THREAD

**Issue by gnerred** [camptocamp/ngeo#2126]
It is not possible to turn the map with fingers on ngeo mobile applications. This is a functionnality that was planned to be integrated in version 2. However this is a core functionnality of OL3.

**Comment by fredj** [comment]
See also https://github.com/camptocamp/c2cgeoportal/issues/1861

**Comment by ybolognini** [comment]
@gnerred 
We had a lot of discussions about this feature. First we decided to deactivate rotation, then we moved it to 2.1 but we never really wrote any specifications. Yes, it's an OL3 functionnality. But we don't think it's a good idea to simply activate it without implementing a "north button".

**Comment by rbovard** [comment]
The "north button" appears by default when the map is rotated, see http://openlayers.org/en/latest/examples/simple.html (as I said on 11st Dec 2015 https://github.com/camptocamp/c2cgeoportal/issues/1861#issuecomment-163976467)

By the way, why the tracking issue for this feature in 2.1 is closed? https://github.com/camptocamp/c2cgeoportal/issues/1938

**Comment by gnerred** [comment]
+1 @rbovard . Why is it not possible to use this default north button "as is" ?

**Comment by ger-benjamin** [comment]
The problem is to translate correctly the tooltip of the button.
Look at: https://github.com/camptocamp/ngeo/issues/919

**Comment by ybolognini** [comment]
@gnerred @rbovard 

First, I must admit I don't know why I closed issue https://github.com/camptocamp/c2cgeoportal/issues/1938 . It was done in May 2016 and I don't remember. Let's continue to discuss this here.

I just talked with @sbrunner, adding this feature would need to think about lots of details like where to put the button, how it will look like, will the rotation be activated by GPS or fingers only, will the desktop include rotation.

We're working hard to fix the last bugs in order to release the 2.1. And it doesn't look like it's simply a matter of activating a standard OL3 feature.

**Comment by kalbermattenm** [comment]
PSC ping @rbovard @gnerred @vvmalbre @mriedo @gsollberger @svavoi @cedricmoullet

I just talked about that with @ybolognini. They are focused on resolving GMF 2.1 bugs in order to be able to release a RC version, thus a final GMF 2.1. Maybe it would be good to hold on a bit with that (no to inject further bugs or issues), because it means adding a new button, rotating the map without altering other things, etc.

I propose to talk about this point at the next PSC meeting (30th January) and I will compile a little summary of the things we already talked about concerning the map rotation.

I hope this suits everybody...
